image_crop/recrop_min_size.py

```diff
@@ -317,7 +346,8 @@ def main():
                 dt -= timeGapDelta
                 for dirName in archiveDirs:
                     logging.warning('Searching for files in dir %s', dirName)
-                    imgPaths = img_archive.getFilesAjax(cookieJar, settings.downloadDir, nameParsed['cameraID'], dirName, dt, dt, 1)
+                    #+##REPLACE DEP. GDRIVE W HPREWN#imgPaths = img_archive.downloadFilesHttp(settings.downloadDir, nameParsed['cameraID'], dirName, dt, dt, 1, 0)
+                    imgPaths = img_archive.getFilesAjax(cookieJar, settings.downloadDir, nameParsed['cameraID'], dirName, dt, dt, 1)#-##REPLACE DEP. GDRIVE W HPREWN#
```

Comment:
You could've left in your change to use the new img_archive.downloadHttp... here.  Anyway, let's get this merged first and then we can improve later.